.github/workflows/misc-migtests.yml

```diff
@@ -59,6 +59,7 @@ jobs:
         run: |
           cd installer_scripts
           yes | ./install-yb-voyager --install-from-local-source --only-pg-support
+          echo "/usr/lib/postgresql/16/bin" >> "$GITHUB_PATH"
```

Comment:
@makalaaneesh why adding path to postgresql 16 binaries here if installing 15?

Reply:
in installer script we install 16, but the machine already had 14 installed. So I'm explicitly adding 16's binaries to the path so that they get picked up by voyager. 

We have to upgrade the PG to 17 as well.